### PR TITLE
Adding a certified chip and loading all nodes regardless of filters a…

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Initializing certified to undefined so all nodes could load at the start regardless of the certified filter and adding a chip for certified nodes
Screenshot from 2023-11-05 14-08-24
### Changes
- Initializing certified variable to undefined in all solutions
- Adding a certified variable to INode interface
- Adding a chip and a chip color to certified nodes

### Related Issues
#1091
### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by zaelgohary — COMMENTED** [review]
Now all certified nodes are being returned even if it's set to false.

image

**Review by mohamedamer453 — COMMENTED** [review]
Selecting a certified node without enabling the certified filter will not update the calculated resource pricing which is supposed to be 25% higher for certified nodes.

- Selecting a certified node without enabling the certified filter

   image

- Selecting the same certified node but with the certified filter enabled.

   image

**Comment by maayarosama** [comment]
> Now all certified nodes are being returned even if it's set to false.
> 
> image

As per a discussion with @AhmedHanafy725 this's in fact the desired behavior.

**Comment by maayarosama** [comment]
> Selecting a certified node without enabling the certified filter will not update the calculated resource pricing which is supposed to be 25% higher for certified nodes.
> 
> * Selecting a certified node without enabling the certified filter
>   image
> * Selecting the same certified node but with the certified filter enabled.
>   image

Added a watch foe the selectedNode so that the calculator can recalculate when it changes.

**Review by zaelgohary — COMMENTED** [review]
Please pull latest changes from base branch as I can't review.
image

**Review by zaelgohary — CHANGES_REQUESTED** [review]
As @mohamedamer453  mentioned, Same node has 2 different prices if the certified chip is enabled.
image

image

**Comment by zaelgohary** [comment]
Can you merge base branch? There was a build error and it was fixed.

**Review by mohamedamer453 — APPROVED** [review]

File: packages/playground/src/weblets/full_vm.vue [hunk 1]
```diff
@@ -230,6 +231,15 @@ watch(
...
 );
+watch(
+  selectedNode,
+  selectedNode => {
+    if (selectedNode) {
+      console.log("selectedNode in fullvm: ", selectedNode);
+    }
+  },
+  { immediate: true },
+);
```
Comment (zaelgohary) [inline packages/playground/src/weblets/full_vm.vue:242]:
```suggestion
```

Can we remove this watcher? It's not needed.

**Review by zaelgohary — CHANGES_REQUESTED** [review]

**Review by zaelgohary — APPROVED** [review]

**Pull request merged**